src/cmd.ts

```diff
@@ -63,6 +63,46 @@ function createCommandWithSharedOptions(name, description) {
 			"Torznab urls with apikey included (separated by spaces)",
 			fallback(fileConfig.torznab)
 		)
+		.option(
+			"-d, --data-dirs <dirs...>",
+			"Directories to use if searching by data instead of torrents (separated by spaces)",
+			fallback(fileConfig.dataDirs)
+		)
+		.option(
+			"-dm, --data-mode <safe, risky>",
+			"Safe will only download torrents with perfect matches. Risky will allow for renames and more matches, but might cause false positives",
+			fallback(fileConfig.dataMode, "safe")
+		)
+		.option(
+			"-dc --dataCategory <cat>",
+			"Category to assign torrents from data-based matching",
+			fallback(fileConfig.dataCategory, "cross-seed-data")
+		)
+		.option(
+			"--link-dir <dir>",
+			"Directory to output data-matched hardlinks to",
+			fileConfig.linkDir
+		)
+		.option(
+			"--use-hardlinks",
+			"Will use hardlinks if a link-dir is supplied instead of symlinks",
+			fallback(fileConfig.useHardlinks, false)
+		)
+		.option(
+			"--skip-recheck",
+			"Skips torrent recheck upon adding to QBittorrent",
+			fallback(fileConfig.skipRecheck, false)
+		)
```

Comment:
We are intelligent about it now for qbit, but I agree without second point. For qbit rn if matching is safe it will skip recheck by default and if matching is risky it will recheck. I set it up this way so users could opt into skipping recheck on risky torrents depending on their risk tolerance

Reply:
would it start downloading the torrents without a recheck? or just have a missing files error or something?

Reply:
Missing files is what happens. I agree about the soft obligation, perhaps a note that this is QBit only is ample? I'm not really sure which of these features exist in other clients.

Reply:
> It's a double negative, but we want skip rechecking off by default.

worth noting that this only applies to data-based searchees. For torrent based searchees, skipping a recheck is the default and doesn't really have any reason to be toggled

I might have asked this before and forgotten the answer, but do we still _need_ a recheck now that we're using links? one problem with rechecking is that if the files are actually missing, it just starts straight up downloading the torrent, which is probably worse than a missing files error. (i've been burned by this before) So if, now that we're using linking, we don't need rechecks (which, theoretically, we shouldn't) then I'd be in favor of nixing this option altogether

Reply:
The rechecking exists because of false positives. It's something I've been wary of though I'm not sure if I've had it happen. When we get into risky matching and allowing name changes we don't have the old guarantee on a perfect match. I've gotten a good few 99.9% rechecks on this, though I'm not sure if a) it's related to risky matching, after all what are the odds the files have *exactly* the same length and b) this wouldn't actually happen with base cross-seed given my hypothesis of it being related to piece size.

Regardless, it only really exists with risky as that's where false positives are possible. I mainly added it as a liability; false positives are possible with risky but if the user opts to not recheck (when a warning tells them they should), then the burden is on them.